cpp/demo/interpolation-io/CMakeLists.txt

```diff
@@ -16,6 +16,8 @@ if (NOT TARGET dolfinx)
   find_package(DOLFINX REQUIRED)
 endif()
 
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
```

Comment:
Why has this been added? 

I don't see that the script that generates the CMake files has been updated.